nodes/ext_container/ext_container.go

```diff
@@ -52,6 +53,8 @@ func (e *extcont) Deploy(ctx context.Context, _ *nodes.DeployParams) error {
 	// set nspath in node config
 	e.Cfg.NSPath = nspath
 
+	e.SetState(state.Deployed)
+
```

Comment:
to verify if this is required for external nodes, or is it being set somewhere else already